### ISSUE TITLE
Insufficient color contrast for buttons

### ISSUE THREAD

**Issue by darekkay** [reakit/reakit#354]
### Current behavior

The contrast ratio between the button's font color (`#fff`) and background color (`rgb(0, 123, 255)`) is not sufficient to pass WCAG AA (see [contrast-ratio](https://contrast-ratio.com/#white-on-rgb%280%2C%20123%2C%20255%29)). 

The same applies to the link color in your documentation.

### Expected behavior

The contrast ratio for buttons (and links) should pass at least level AA (4.5).

### Possible solutions

Use a WCAG-compliant blue color. I also like a light blue color, so personally I first went with `#006aff`. However, this won't pass AA on a light grey background, so I switched to `#005aff` and I'm using it for most projects ever since.

**Comment by diegohaz** [comment]
Thank you for bringing this up @darekkay. For theming, I simply borrowed most of the styles from bootstrap. But this does need more attention.

If anyone is interested in helping, these are the relevant files:

`reakit-system-palette`:
https://github.com/reakit/reakit/blob/master/packages/reakit-system-palette/src/palette.ts

website theme:
https://github.com/reakit/reakit/blob/master/packages/website/src/components/Provider.tsx

**Comment by MateusAndrade** [comment]
> Thank you for bringing this up @darekkay. For theming, I simply borrowed most of the styles from bootstrap. But this does need more attention.
> 
> If anyone is interested in helping, these are the relevant files:
> 
> `reakit-system-palette` (used by examples):
> https://github.com/reakit/reakit/blob/master/packages/reakit-system-palette/src/palette.ts
> 
> website theme:
> https://github.com/reakit/reakit/blob/master/packages/website/src/components/Provider.tsx

Im going to give a look at this. =D

**Comment by darekkay** [comment]
> For theming, I simply borrowed most of the styles from bootstrap.

This is the reason why I don't understand Bootstrap's [decision](https://getbootstrap.com/docs/4.1/getting-started/accessibility/) to use inaccessible colors on purpose:

> Most colors that currently make up Bootstrap’s default palette—used throughout the framework for things such as button variations, alert variations, form validation indicators **lead to insufficient color contrast** (below the recommended WCAG 2.0 color contrast ratio of 4.5:1) when used against a light background. Authors will need to manually modify/extend these default colors to ensure adequate color contrast ratios.
